statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java

```diff
@@ -354,6 +347,42 @@ private void registerAdd() {
         });
     }
 
+    /*
+     * computes no. of days from the given date till the current date.
+     * {{computeDays dateNode 'dd MM yyyy'}}
+     */
+    private void registerComputeDays() {
+        handlebars.registerHelper("computeDays", (String context, Options options) -> {
+            try {
+                if (null == options.params || options.params.length == 0) {
+                    return 0L;
+                }
+                if (null == context || Strings.isNullOrEmpty(context)) {
+                    return 0L;
+                }
+                String dateFormat = options.param(0);
+                if(dateFormat == null || Strings.isNullOrEmpty(dateFormat)) {
+                    return 0L;
+                }
+                SimpleDateFormat sdf = new SimpleDateFormat(dateFormat);
+                String timeZone =
+                        options.params.length < 2
+                                ? DEFAULT_TIME_ZONE
+                                : options.param(1);
+                sdf.setTimeZone(TimeZone.getTimeZone(timeZone));
+                val epochTimestamp = sdf.parse(context).getTime();
+                val diff = System.currentTimeMillis() - epochTimestamp;
+                if(diff < 0) {
+                    return 0L;
+                }
+                return Math.round(diff / (1000 * 60 * 60 * 24));
```

Comment:
Evaluate `1000 * 60 * 60 * 24` in a constant. Its actually 86_400_000.

Reply:
done.